www/assets/css/course-collection.scss

```diff
@@ -41,7 +41,7 @@ h1.course-list-title {
     padding: 15px;
   }
 
-  .collection-description {
+  .collection-content {
     padding: 15px;
   }
```

Comment:
In one place, in `course-v2/layouts/lists/single.html` we are still using `collection-description` class. But it's for readability purposes only. The styling never applied there as this is `www` theme.